uaclient/api/u/unattended_upgrades/status/v1.py

```diff
@@ -40,12 +40,22 @@ def __init__(self, msg: str, code: str):
 
 class UnattendedUpgradesStatusResult(DataObject, AdditionalInfo):
     fields = [
-        Field("systemd_apt_timer_enabled", BoolDataValue),
-        Field("apt_periodic_job_enabled", BoolDataValue),
-        Field("package_lists_refresh_frequency_days", IntDataValue),
-        Field("unattended_upgrades_frequency_days", IntDataValue),
-        Field("unattended_upgrades_allowed_origins", StringDataValue),
-        Field("unattended_upgrades_running", BoolDataValue),
+        Field("systemd_apt_timer_enabled", BoolDataValue, required=False),
+        Field("apt_periodic_job_enabled", BoolDataValue, required=False),
+        Field(
+            "package_lists_refresh_frequency_days",
+            IntDataValue,
+            required=False,
+        ),
+        Field(
+            "unattended_upgrades_frequency_days", IntDataValue, required=False
+        ),
+        Field(
+            "unattended_upgrades_allowed_origins",
+            StringDataValue,
```

Comment:
unrelated but something to fix: 
```suggestion
            data_list(StringDataValue),
```